File: DemoMethods/Entities/LastFM.cs
using System;
using System.Collections.Generic;

namespace DemoMethods.Entities
{
    public class LastFm
    {
        public string Artist { get; set; }
        public DateTime TimeStamp { get; set; }
        public List<List<string>> Similars { get; set; }
        public List<List<string>> Tags { get; set; }
        public string Track_Id { get; set; }
        public string Title { get; set; }
    }
}

File: DemoMethods/src/main/java/net/ravendb/demo/entities/LastFm.java
package net.ravendb.demo.entities;

import com.mysema.query.annotations.QueryEntity;

import java.util.Date;
import java.util.List;

@QueryEntity
public class LastFm {
    private String artist;
    private String trackId;
    private String title;
    private Date timeStamp;
    private List<String> tags;

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public String getTrackId() {
        return trackId;
    }

    public void setTrackId(String trackId) {
        this.trackId = trackId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Date getTimeStamp() {
        return timeStamp;
    }

    public void setTimeStamp(Date timeStamp) {
        this.timeStamp = timeStamp;
    }

    public List<String> getTags() {
        return tags;
    }

    public void setTags(List<String> tags) {
        this.tags = tags;
    }
}
